extensions/amp-inline-gallery/0.1/amp-inline-gallery-thumbnails.css

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+amp-inline-gallery-thumbnails .i-amphtml-carousel-content {
+  padding: 0;
+  padding: 0 calc(var(--thumbnail-margin, 0) / 2);
+}
+
+.i-amphtml-inline-gallery-thumbnails-container  {
+  box-sizing: border-box;
+  width: auto !important;
+  height: 100%;
+  flex-basis: content;
+  padding: 0;
+  padding: var(--thumbnail-margin, 0) calc(var(--thumbnail-margin, 0) / 2);
+  /** Used for responsive sizing based on height. */
+  writing-mode: vertical-lr;
+}
+
+.i-amphtml-inline-gallery-thumbnails-thumbnail {
+  position: relative;
+  width: auto;
+  height: 100%;
+}
+
+.i-amphtml-inline-gallery-thumbnails-resizer {
+  position: static;
+  height: 100%;
+}
+
+.i-amphtml-inline-gallery-thumbnails-image,
+.i-amphtml-inline-gallery-thumbnails-default {
+  position: absolute;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+}
+
+.i-amphtml-inline-gallery-thumbnails-default {
+  background-color: #999;
```

Comment:
We should make this customizable

Reply:
Can deal with multiple different aspects of customization (including selection state) in a follow up PR.